Automattic/wp-calypso
# Jetpack NUX: Update error screen for plugins auto-config

We should show more detailed error messages when we  can't install plugins, or when installation fails.

To test (part 1):

1. Visit the install page for a site with `DISALLOW_FILE_MODS` set, or which is a subsite of a multisite, or any other case we can't install plugins on.
2. You'll see an "Oh no!" message, with text saying why we can't install plugins on this site.
3. You can click a "Contact Support" button.

Preview:

![screen shot 2016-06-30 at 6 06 47 pm](https://cloud.githubusercontent.com/assets/541093/16525154/0da22120-3f79-11e6-8ce7-da071a0c9bb3.png)

To test (part 2):

1. Visit the install page with a site we know will encounter an error (easy breakage: create a folder called `akismet` in your plugins folder, set permissions so the web user can't write to it)
2. Your plugins will start to install, then encounter whatever error case you have. The plugin with the error gets a compact error notice.
3. When all plugins are installed, a larger notice appears, saying that there was an error, with a link to manual install instructions, and an action to contact support.

Preview:

![screen shot 2016-07-01 at 5 15 56 pm](https://cloud.githubusercontent.com/assets/541093/16534685/bbd9b3da-3faf-11e6-9f07-03f19e419f5f.png)

Fixes #6432

Test live: https://calypso.live/?branch=update/jetpack-config-errors
